module/PowerShellEditorServices/Commands/Public/CmdletInterface.ps1

```diff
@@ -136,6 +136,11 @@ function New-EditorFile {
                         $preview = $true
                     }
 
+                    # Resolve full path before passing to editor
+                    if (!([System.IO.Path]::IsPathRooted($fileName))) {
+                        $fileName = (Resolve-Path $fileName).Path
```

Comment:
Assume `$fileName` is guaranteed to exist?

(Since:)
```
> Resolve-Path ./nowhere
Resolve-Path : Cannot find path 'C:\Users\roholt\nowhere' because it does not exist.
At line:1 char:1
+ Resolve-Path ./nowhere
+ ~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : ObjectNotFound: (C:\Users\roholt\nowhere:String) [Resolve-Path], ItemNotFoundException
+ FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.ResolvePathCommand
```

Reply:
Otherwise the way to go is:

```powershell
$fileName = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($fileName)
```

Example:

```
> $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath('./nowhere')
C:\Users\roholt\nowhere
```

Reply:
Yes it's guaranteed to exist. See line 131:

```pwsh
New-Item -Path $fileName -ItemType File | Out-Null
```

Reply:
made the change anyway